### PR TITLE
Resource preset

### PR DESCRIPTION
**Pull request by keirthana**
First round of changes for deprecating instance type to denote resources. I have also introduced the new terminology resource preset in some places so that users get comfortable to it. 

There are more mentions in code blocks and command outputs, I will do another round of updates for those.

We should also document a default resource preset - once we full remove `instance-type`, will the `resources` attribute be made mandatory? If not, what will be the resource allocation when no resources are mentioned?

### CODE REVIEW

File: tut/getting-started.md [hunk 1]
```diff
@@ -177,16 +179,18 @@ Complete the following steps to create an application from an APK:
...
-2. Create a `manifest.yaml` file in that folder. The manifest contains the name and [instance type](https://discourse.ubuntu.com/t/application-manifest/24197#instance-type) for the application. Choose an instance type that is suitable for your application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, select an instance type with GPU support like `g4.3`. For other instance types, the container will use a GPU if available or software encoding otherwise.
+2. Create a `manifest.yaml` file in that folder. The manifest contains the name and if you want to define a resource preset, the resources for the application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, make sure to mention the GPU requirement using the `resources` attribute. Otherwise, the container will use a GPU if available or software encoding.
 
    For example, the file could look like this:
 
    ```yaml
    name: my-application
-   instance-type: a4.3
+   resources:
+     cpus: 6
```
Comment (adglkh) [inline tut/getting-started.md:189]: If people create an AMS application with resources preset, the following attributes must be set
- cpus
- memory
- disk-size

In the above case, the `memory` is missing.

File: tut/getting-started.md [hunk 2]
```diff
@@ -60,11 +60,13 @@ Let's start exploring what Anbox Cloud can do by launching a virtual device that
...
 Complete the following steps to create a virtual device:
 
-1. Create a simple `manifest.yaml` file that contains the name and instance type for the virtual device. For example, the file could look like this:
+1. Create a simple `manifest.yaml` file that contains the name and resource preset for the virtual device. For example, the file could look like this:
 
    ```yaml
    name: virtual-device-cli
-   instance-type: a4.3
+   resources:
+     cpus: 4
```
Comment (adglkh) [inline tut/getting-started.md:68]: See the comments below.

File: tut/getting-started.md [hunk 3]
```diff
@@ -177,16 +179,18 @@ Complete the following steps to create an application from an APK:
 
 1. Create a folder for your application (for example, `my-application`) and place your APK inside this folder.
 
-2. Create a `manifest.yaml` file in that folder. The manifest contains the name and [instance type](https://discourse.ubuntu.com/t/application-manifest/24197#instance-type) for the application. Choose an instance type that is suitable for your application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, select an instance type with GPU support like `g4.3`. For other instance types, the container will use a GPU if available or software encoding otherwise.
+2. Create a `manifest.yaml` file in that folder. The manifest contains the name and if you want to define a resource preset, the resources for the application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, make sure to mention the GPU requirement using the `resources` attribute. Otherwise, the container will use a GPU if available or software encoding.
```
Comment (adglkh) [inline tut/getting-started.md:182]: The manifest contains the name and if you want to define a resource preset, the resources for the application
->
This manifest contains the application's name and, if necessary, define resource presets for the application. By default, the following resource presets will be used if they're not defined
- cpus: 2
- memory: 3GB
- disk-size: 3GB

File: exp/capacity-planning.md [hunk 4]
```diff
@@ -17,18 +17,16 @@ When estimating capacity, consider the following questions to better understand
...
-The instance type specifies the resources that are available to the application:
+A default resource preset will be set for every application. A resource preset specifies the resources that are available to the application:
 
 - The number of vCPU cores
 - The amount of RAM
 - The amount of disk space
 - The number of GPU slots
 
-If none of the provided instance types fits for your application, you can also manually [configure the resources](https://discourse.ubuntu.com/t/how-to-configure-available-resources/24960) according to your requirements.
+Depending on the resources that your application requires, if the default resource preset does not suit, you can choose suitable [resources](https://discourse.ubuntu.com/t/application-manifest/24197#resources) that fit your application.
```
Comment (adglkh) [inline exp/capacity-planning.md:27]: Just wondering where can people refer to the default resource presets.

**Review by adglkh — CHANGES_REQUESTED** [review]

File: tut/getting-started.md (hunk 3 above)
Comment (morphis) [reply tut/getting-started.md:182]: The disk size needs to be differentiated base on if its a VM or a container. For containers we set 15GB currently as the default

**Review by morphis — CHANGES_REQUESTED** [review]

File: tut/getting-started.md (hunk 1 above)
Comment (keirthana) [reply tut/getting-started.md:192]: Added now

File: tut/getting-started.md (hunk 2 above)
Comment (keirthana) [reply tut/getting-started.md:68]: Added now

File: exp/capacity-planning.md (hunk 4 above)
Comment (keirthana) [reply exp/capacity-planning.md]: Linked to the topic

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**